src/test_typing_extensions.py

```diff
@@ -5712,7 +5712,6 @@ class Y(Generic[T], NamedTuple):
                 self.assertEqual(a.x, 3)
 
                 things = "arguments" if sys.version_info >= (3, 10) else "parameters"
```

Comment:
Seems like these lines got re-added -- but maybe they were breaking for 3.8/3.9?

Reply:
I think so, yes. I see you commented on this in https://github.com/python/typing_extensions/pull/293#discussion_r1375254093 before; I don't feel too strongly but I think it's best to match CPython's behavior.